src/index.html

```diff
@@ -1,13 +1,370 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Miami</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@300&family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="conteiner">
+        <div class="header__content">
+          <div class="top-actions">
+            <a href="#" class="logo">
+              <img 
+                src="images/MyBiKE.png" 
+                alt="MyBiKE"
+                class="logo__images"
+              >
+            </a> 
+            <a href="tell:+1 234 5555-55-55" class="icon icon--phone header__call"></a>
```

Comment:
there is a typo, the phone does not ring

<a href="tell:+1 234 5555-55-55"

Reply:
thx